content/en/synthetics/troubleshooting/_index.md

```diff
@@ -28,43 +28,43 @@ If you see a sudden spike or overall increase in your API test [timing metrics][
 
 #### My website is not loading in the iframe
```

Comment:
We should update headers in this docs as they use `My` `I`. Looks like this was never updated when we moved away from Q&A-style headings.

Reply:
Created DOCS-2974 to track this doc work!